RSS.PPA

```diff
@@ -174,7 +174,7 @@ alias _rssget {
                   while (%child <= %children) {
                     var %tagname $comval(%com3,%child,tagName)
                     if (%child == 1) $iif($window($2),aline $2,echo -sg) $asctime($timestampfmt) $:::
```

Comment:
Ah cool. Thanks for looking through the code! $asctime($timestampfmt) == $timestamp nice catch. But I think you've missed this occurrence. Could you make the change here as well?